blocks/README.md

```diff
@@ -40,3 +40,27 @@ blocks/
 ```
 
 Please refer to the official [BEM](https://en.bem.info/methodology/quick-start/#introduction) or [BEM Nested](https://en.bem.info/methodology/quick-start/#file-structure) documentation for details if needed.
+
+### Using media queries
```

Comment:
@NazarPauk What do you think about revising the document's structure to improve its readability?

We can add a second-level heading titled _"Files and Folders Structure"_ before line 3 and make _"Using media queries"_ a second-level heading too. This should make it easier to navigate and understand.

Reply:
Do you mean to document the nesting of folders and files using the BEM Nested methodology in a document?

Reply:
@NazarPauk I'm sorry, I am not sure if I got your question right.. we got two parts to this README: BEM Nested and media requests. I propose to add second level heading _Files and Folders Structure_ on line 3. And also to make _Using media queries_ second-level heading.

Reply:
@NazarPauk, please let me know if you want to hop on a quick call so I can explain it better. It's really just a tiny 5-second adjustment here 😉